src/accidental.ts

```diff
@@ -526,17 +562,17 @@ export class Accidental extends Modifier {
     L('Rendering: ', type, accX, accY);
 
     if (!cautionary) {
-      glyph.render(context, accX, accY);
+      glyph?.render(ctx, accX, accY);
```

Comment:
These `?` in this section are all potential silent failures. Can we explicitly check for them? (Or better still not make them optional?)

Reply:
Done.